tests/ProxyManagerTest/Functional/LazyLoadingValueHolderFunctionalTest.php

```diff
@@ -280,7 +280,7 @@ public function testWillAllowMultipleProxyInitialization() : void
 
         $proxy = (new LazyLoadingValueHolderFactory())->createProxy(
             BaseClass::class,
-            static function (?object & $wrappedInstance) use (& $counter) : bool {
+            static function (?BaseClass & $wrappedInstance) use (& $counter) : bool {
```

Comment:
This should still be valid with `?object`.

In fact, in libraries like `doctrine/orm`, this will work with numerous types of objects

Reply:
Yeah, I think these should actually both have the same error: https://psalm.dev/r/1785fafb65